base/sysimg.jl

```diff
@@ -313,6 +309,11 @@ include("lock.jl")
 include("threads.jl")
 include("weakkeydict.jl")
 
+# Logging
+include("logging.jl")
+using .CoreLogging
+global_logger(SimpleLogger(Core.STDERR, CoreLogging.Info))
```

Comment:
I think the assignment to `_global_logstate` can just be done at the end of the `logging.jl` file, and then also drop the `using` statement

Reply:
The `using` (or an import) is required to bring the logging macro names into `Base` for later use in the sysimg, for example in `loading.jl`.  So I'll leave this here unless there's a particular preference to do that with an import.

I set the logger here in sysimg because the value we're setting for the global logger is entirely related to bootstrap, not to the logging system per se. So I'd rather leave it in `sysimg.jl` because that's the place where the bootstrap ordering is defined.